source/gx/tilix/terminal/password.d

```diff
@@ -285,7 +285,7 @@ private:
 
     HashTable createHashTable() {
         import gtkc.glib;
-        return new HashTable(g_str_hash, g_str_equal);
+        return new HashTable(&g_str_hash, &g_str_equal);
```

Comment:
Use `ref` instead please.

https://en.wikibooks.org/wiki/D_(The_Programming_Language)/d2/Pointers,_Pass-By-Reference_and_Static_Arrays#Pass_By_Reference

Reply:
ref is a keyword in function signatures, not in calls??

Reply:
Yes, if you need this then do it in the function call. I assume this error will propagate to other calls, right? Or is it context specific? If then then this is fine.

Reply:
HashTable is obviously not a Tilix type, the constructor we're calling here is GtkD's wrapper around `g_hash_table_new`, we're giving it pointers to C functions (which `g_str_hash` and `_equal` are)

Reply:
So it seems this is the correct approach, I wish GtkD would provide more type safety.